### DISCUSSION TITLE
Vulnerability scanner? (where can I find it?)

### DISCUSSION THREAD

**Discussion by lacak-sk** [utmstack/UTMStack#640] [Q&A] [answered]
Hi, as mentioned on https://docs.utmstack.com/ there should be in UTMStack also Vulnerability Scanner. But where can I find it? In top/right menu I do not see such item ...?
(Version 10.3.0 FREE)

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @lacak-sk Vulnerability Scanner is a Standalone product, and is not part of the UTMStack platform. We will release a documentation fix shortly. Thanks for the feed back.

**Reply by stepcellwolf** [reply]
Any updates on this and vulnerability scanner?

**Comment by lacak-sk** [comment]
Thanks, now I found that it is mentioned in "(provided as a standalone application)" on https://docs.utmstack.com/ in the beginning of the page.
But later is there stand-alone paragraph "Vulnerability Scanner" which gives false impression that it is part of UTMStack.
